CONTRIBUTING.md

```diff
@@ -89,5 +99,6 @@ Every lint rule has an ID which can be used in lint directives (comments beginni
 * See [here](https://docs.rs/wdl/latest/wdl/grammar/type.SyntaxNode.html) for the `SyntaxNode` docs.
 * The PR which introduced `exceptable_nodes()` and `exceptable_add()` is [#162](https://github.com/stjude-rust-labs/wdl/pull/162).
 * That PR fixed issue [#135](https://github.com/stjude-rust-labs/wdl/issues/135)
+* The PR which introduced `related_rules()` is [#371](https://github.com/stjude-rust-labs/wdl/pull/371))
```

Comment:
GitHub won't let me comment on the right line.
But I think this good to add, but the `#### Further reading` header should be bumped back to `##` if we're including this.

Currently, the `Further reading` section is nested beneath https://github.com/stjude-rust-labs/wdl/blob/main/CONTRIBUTING.md#can-you-explain-how-rules-use-exceptable_nodes-and-exceptable_add as all the bullet points relate to that Q. With your addition we should bump out this section to be its own section taking place after the FAQs